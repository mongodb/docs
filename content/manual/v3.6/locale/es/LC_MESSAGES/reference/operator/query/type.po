# SOME DESCRIPTIVE TITLE.
# Copyright (C) 2011-2014, MongoDB, Inc.
# This file is distributed under the same license as the mongodb-manual
# package.
#
# Translators:
msgid ""
msgstr ""
"Project-Id-Version: MongoDB Manual\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2019-03-19 11:30-0400\n"
"PO-Revision-Date: 2014-04-08 18:55+0000\n"
"Last-Translator: tychoish <tychoish@gmail.com>\n"
"Language: es\n"
"Language-Team: Spanish (http://www.transifex.com/projects/p/mongodb-"
"manual/language/es/)\n"
"Plural-Forms: nplurals=2; plural=(n != 1)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 2.6.0\n"

#: ../source/reference/operator/query/type.txt:3
msgid "$type"
msgstr ""

#: ../source/reference/operator/query/type.txt:14
msgid "Definition"
msgstr ""

#: ../source/reference/operator/query/type.txt:18
msgid ""
":query:`$type` selects the documents where the *value* of the ``field`` "
"is an instance of the specified :term:`BSON` type(s). Querying by data "
"type is useful when dealing with highly unstructured data where data "
"types are not predictable."
msgstr ""

#: ../source/reference/operator/query/type.txt:23
msgid ""
"A :query:`$type` expression for a single :term:`BSON` type has the "
"following syntax:"
msgstr ""

#: ../source/reference/operator/query/type.txt:32
msgid ""
"You can specify either the number or alias for the :ref:`BSON type"
"<document-type-available-types>`"
msgstr ""

#: ../source/reference/operator/query/type.txt:35
msgid ""
"The :query:`$type` expression can also accept an array of :term:`BSON` "
"types and has the following syntax:"
msgstr ""

#: ../source/reference/operator/query/type.txt:42
msgid ""
"The above query will match documents where the ``field`` value is any of "
"the listed types. The types specified in the array can be either numeric "
"or string aliases."
msgstr ""

#: ../source/reference/operator/query/type.txt:46
msgid "See :ref:`document-querying-by-multiple-data-types` for an example."
msgstr ""

#: ../source/reference/operator/query/type.txt:48
msgid ""
":ref:`document-type-available-types` describes the BSON types and their "
"corresponding numeric and string aliases."
msgstr ""

#: ../source/reference/operator/query/type.txt:53
msgid ""
"If you wish to obtain the :ref:`BSON type <bson-types>` returned by an "
":ref:`operator expression <agg-quick-ref-operator-expressions>` rather "
"than filtering documents by their BSON type, use the :expression:`$type` "
"aggregation operator."
msgstr ""

#: ../source/reference/operator/query/type.txt:59
msgid "Behavior"
msgstr ""

#: ../source/reference/operator/query/type.txt:61
msgid ""
":query:`$type` returns documents where the BSON type of the ``field`` "
"matches the BSON type passed to :query:`$type`."
msgstr ""

#: ../source/reference/operator/query/type.txt:66
msgid ""
":query:`$type` now works with arrays in the same way it works with other "
"BSON types. Previous versions only matched documents where the ``field`` "
"contained a nested array."
msgstr ""

#: ../source/reference/operator/query/type.txt:73
msgid "Available Types"
msgstr ""

#: ../source/reference/operator/query/type.txt:75
msgid ""
":query:`$type` operator accepts string aliases for the BSON types in "
"addition to the numbers corresponding to the BSON types. Previous "
"versions only accepted the numbers corresponding to the BSON type."
msgstr ""

#: ../source/includes/fact-bson-types.rst:4
msgid "Type"
msgstr ""

#: ../source/includes/fact-bson-types.rst:5
msgid "Number"
msgstr ""

#: ../source/includes/fact-bson-types.rst:6
msgid "Alias"
msgstr ""

#: ../source/includes/fact-bson-types.rst:7
msgid "Notes"
msgstr ""

#: ../source/includes/fact-bson-types.rst:9
msgid "Double"
msgstr ""

#: ../source/includes/fact-bson-types.rst:10
msgid "1"
msgstr ""

#: ../source/includes/fact-bson-types.rst:11
msgid "\"double\""
msgstr ""

#: ../source/includes/fact-bson-types.rst:14
msgid "String"
msgstr ""

#: ../source/includes/fact-bson-types.rst:15
msgid "2"
msgstr ""

#: ../source/includes/fact-bson-types.rst:16
msgid "\"string\""
msgstr ""

#: ../source/includes/fact-bson-types.rst:19
msgid "Object"
msgstr ""

#: ../source/includes/fact-bson-types.rst:20
msgid "3"
msgstr ""

#: ../source/includes/fact-bson-types.rst:21
msgid "\"object\""
msgstr ""

#: ../source/includes/fact-bson-types.rst:24
msgid "Array"
msgstr ""

#: ../source/includes/fact-bson-types.rst:25
msgid "4"
msgstr ""

#: ../source/includes/fact-bson-types.rst:26
msgid "\"array\""
msgstr ""

#: ../source/includes/fact-bson-types.rst:29
msgid "Binary data"
msgstr ""

#: ../source/includes/fact-bson-types.rst:30
msgid "5"
msgstr ""

#: ../source/includes/fact-bson-types.rst:31
msgid "\"binData\""
msgstr ""

#: ../source/includes/fact-bson-types.rst:34
msgid "Undefined"
msgstr ""

#: ../source/includes/fact-bson-types.rst:35
msgid "6"
msgstr ""

#: ../source/includes/fact-bson-types.rst:36
msgid "\"undefined\""
msgstr ""

#: ../source/includes/fact-bson-types.rst:37
#: ../source/includes/fact-bson-types.rst:67
#: ../source/includes/fact-bson-types.rst:77
msgid "Deprecated."
msgstr ""

#: ../source/includes/fact-bson-types.rst:39
msgid "ObjectId"
msgstr ""

#: ../source/includes/fact-bson-types.rst:40
msgid "7"
msgstr ""

#: ../source/includes/fact-bson-types.rst:41
msgid "\"objectId\""
msgstr ""

#: ../source/includes/fact-bson-types.rst:44
msgid "Boolean"
msgstr ""

#: ../source/includes/fact-bson-types.rst:45
msgid "8"
msgstr ""

#: ../source/includes/fact-bson-types.rst:46
msgid "\"bool\""
msgstr ""

#: ../source/includes/fact-bson-types.rst:49
msgid "Date"
msgstr ""

#: ../source/includes/fact-bson-types.rst:50
msgid "9"
msgstr ""

#: ../source/includes/fact-bson-types.rst:51
msgid "\"date\""
msgstr ""

#: ../source/includes/fact-bson-types.rst:54
msgid "Null"
msgstr ""

#: ../source/includes/fact-bson-types.rst:55
msgid "10"
msgstr ""

#: ../source/includes/fact-bson-types.rst:56
msgid "\"null\""
msgstr ""

#: ../source/includes/fact-bson-types.rst:59
msgid "Regular Expression"
msgstr ""

#: ../source/includes/fact-bson-types.rst:60
msgid "11"
msgstr ""

#: ../source/includes/fact-bson-types.rst:61
msgid "\"regex\""
msgstr ""

#: ../source/includes/fact-bson-types.rst:64
msgid "DBPointer"
msgstr ""

#: ../source/includes/fact-bson-types.rst:65
msgid "12"
msgstr ""

#: ../source/includes/fact-bson-types.rst:66
msgid "\"dbPointer\""
msgstr ""

#: ../source/includes/fact-bson-types.rst:69
msgid "JavaScript"
msgstr ""

#: ../source/includes/fact-bson-types.rst:70
msgid "13"
msgstr ""

#: ../source/includes/fact-bson-types.rst:71
msgid "\"javascript\""
msgstr ""

#: ../source/includes/fact-bson-types.rst:74
msgid "Symbol"
msgstr ""

#: ../source/includes/fact-bson-types.rst:75
msgid "14"
msgstr ""

#: ../source/includes/fact-bson-types.rst:76
msgid "\"symbol\""
msgstr ""

#: ../source/includes/fact-bson-types.rst:79
msgid "JavaScript (with scope)"
msgstr ""

#: ../source/includes/fact-bson-types.rst:80
msgid "15"
msgstr ""

#: ../source/includes/fact-bson-types.rst:81
msgid "\"javascriptWithScope\""
msgstr ""

#: ../source/includes/fact-bson-types.rst:84
#: ../source/reference/operator/query/type.txt:88
msgid "32-bit integer"
msgstr ""

#: ../source/includes/fact-bson-types.rst:85
msgid "16"
msgstr ""

#: ../source/includes/fact-bson-types.rst:86
msgid "\"int\""
msgstr ""

#: ../source/includes/fact-bson-types.rst:89
msgid "Timestamp"
msgstr ""

#: ../source/includes/fact-bson-types.rst:90
msgid "17"
msgstr ""

#: ../source/includes/fact-bson-types.rst:91
msgid "\"timestamp\""
msgstr ""

#: ../source/includes/fact-bson-types.rst:94
#: ../source/reference/operator/query/type.txt:90
msgid "64-bit integer"
msgstr ""

#: ../source/includes/fact-bson-types.rst:95
msgid "18"
msgstr ""

#: ../source/includes/fact-bson-types.rst:96
msgid "\"long\""
msgstr ""

#: ../source/includes/fact-bson-types.rst:99
msgid "Decimal128"
msgstr ""

#: ../source/includes/fact-bson-types.rst:100
msgid "19"
msgstr ""

#: ../source/includes/fact-bson-types.rst:101
msgid "\"decimal\""
msgstr ""

#: ../source/includes/fact-bson-types.rst:102
msgid "New in version 3.4."
msgstr ""

#: ../source/includes/fact-bson-types.rst:104
msgid "Min key"
msgstr ""

#: ../source/includes/fact-bson-types.rst:105
msgid "-1"
msgstr ""

#: ../source/includes/fact-bson-types.rst:106
msgid "\"minKey\""
msgstr ""

#: ../source/includes/fact-bson-types.rst:109
msgid "Max key"
msgstr ""

#: ../source/includes/fact-bson-types.rst:110
msgid "127"
msgstr ""

#: ../source/includes/fact-bson-types.rst:111
msgid "\"maxKey\""
msgstr ""

#: ../source/reference/operator/query/type.txt:83
msgid ""
":query:`$type` supports the ``number`` alias, which will match against "
"the following :term:`BSON` types:"
msgstr ""

#: ../source/reference/operator/query/type.txt:86
msgid "double"
msgstr ""

#: ../source/reference/operator/query/type.txt:92
msgid "decimal"
msgstr ""

#: ../source/reference/operator/query/type.txt:94
msgid "See :ref:`document-querying-by-data-type`"
msgstr ""

#: ../source/reference/operator/query/type.txt:97
msgid "MinKey and MaxKey"
msgstr ""

#: ../source/reference/operator/query/type.txt:99
msgid ""
":bsontype:`MinKey <data_minkey>` and :bsontype:`MaxKey <data_maxkey>` are"
" used in comparison operations and exist primarily for internal use. For "
"all possible :term:`BSON` element values, ``MinKey`` will always be the "
"smallest value while ``MaxKey`` will always be the greatest value."
msgstr ""

#: ../source/reference/operator/query/type.txt:104
msgid ""
"Querying for  ``minKey`` or ``maxKey`` with :query:`$type` will only "
"return fields that match the special ``MinKey`` or ``MaxKey`` values."
msgstr ""

#: ../source/reference/operator/query/type.txt:108
msgid ""
"Suppose that the ``data`` collection has two documents with ``MinKey`` "
"and ``MaxKey``:"
msgstr ""

#: ../source/reference/operator/query/type.txt:116
msgid "The following query will return the document with ``_id: 1``:"
msgstr ""

#: ../source/reference/operator/query/type.txt:122
msgid "The following query will return the document with ``_id: 2``:"
msgstr ""

#: ../source/reference/operator/query/type.txt:129
msgid "Examples"
msgstr ""

#: ../source/reference/operator/query/type.txt:134
msgid "Querying by Data Type"
msgstr ""

#: ../source/reference/operator/query/type.txt:136
msgid ""
"The ``addressBook`` contains addresses and zipcodes, where ``zipCode`` "
"has ``string``, ``int``, ``double``, and ``long`` values:"
msgstr ""

#: ../source/reference/operator/query/type.txt:151
msgid ""
"The following queries return all documents where ``zipCode`` is the "
":term:`BSON` type ``string``:"
msgstr ""

#: ../source/reference/operator/query/type.txt:160
#: ../source/reference/operator/query/type.txt:174
#: ../source/reference/operator/query/type.txt:187
msgid "These queries return:"
msgstr ""

#: ../source/reference/operator/query/type.txt:166
msgid ""
"The following queries return all documents where ``zipCode`` is the "
":term:`BSON` type ``double``:"
msgstr ""

#: ../source/reference/operator/query/type.txt:180
msgid ""
"The following query uses the ``number`` alias to return documents where "
"``zipCode`` is the :term:`BSON` type ``double``, ``int``, or ``long``:"
msgstr ""

#: ../source/reference/operator/query/type.txt:198
msgid "Querying by Multiple Data Type"
msgstr ""

#: ../source/reference/operator/query/type.txt:200
msgid ""
"The ``grades`` collection contains names and averages, where "
"``classAverage`` has ``string``, ``int``, and ``double`` values:"
msgstr ""

#: ../source/reference/operator/query/type.txt:214
msgid ""
"The following queries return all documents where ``classAverage`` is the "
":term:`BSON` type ``string`` or ``double``. The first query uses numeric "
"aliases while the second query uses string aliases."
msgstr ""

#: ../source/reference/operator/query/type.txt:223
msgid "These queries return the following documents:"
msgstr ""

#: ../source/reference/operator/query/type.txt:234
msgid "Querying by MinKey and MaxKey"
msgstr ""

#: ../source/reference/operator/query/type.txt:236
msgid ""
"The ``restaurants`` collection uses ``minKey`` for any grade that is a "
"failing grade:"
msgstr ""

#: ../source/reference/operator/query/type.txt:261
msgid "And ``maxKey`` for any grade that is the highest passing grade:"
msgstr ""

#: ../source/reference/operator/query/type.txt:285
msgid ""
"The following query returns any restaurant whose ``grades.grade`` field "
"contains ``minKey``:"
msgstr ""

#: ../source/reference/operator/query/type.txt:294
#: ../source/reference/operator/query/type.txt:328
msgid "This returns"
msgstr ""

#: ../source/reference/operator/query/type.txt:318
msgid ""
"The following query returns any restaurant whose ``grades.grade`` field "
"contains ``maxKey``:"
msgstr ""

#: ../source/reference/operator/query/type.txt:355
msgid "Querying by Array Type"
msgstr ""

#: ../source/reference/operator/query/type.txt:357
msgid "A collection named ``SensorReading`` contains the following documents:"
msgstr ""

#: ../source/reference/operator/query/type.txt:396
msgid ""
"The following query returns any document in which the ``readings`` field "
"is an array, empty or non-empty."
msgstr ""

#: ../source/reference/operator/query/type.txt:403
msgid "The above query returns the following documents:"
msgstr ""

#: ../source/reference/operator/query/type.txt:438
msgid ""
"In the documents with ``_id : 1``, ``_id : 2``, ``_id : 3``, and ``_id : "
"4``, the ``readings`` field is an array."
msgstr ""

#: ../source/reference/operator/query/type.txt:442
msgid "Additional Information"
msgstr ""

#: ../source/reference/operator/query/type.txt:444
msgid ""
":method:`~db.collection.find()`, :doc:`BSON Types </reference/bson-"
"types>`."
msgstr ""

# 23e9e551a2ad441ca9e1a50d260c74ec
#~ msgid "*Syntax*: ``{ field: { $type: <BSON type> } }``"
#~ msgstr ""

# fa3d55241c7048cdaaad0fc6609d1399
#~ msgid ""
#~ ":query:`$type` selects the documents where "
#~ "the *value* of the ``field`` is "
#~ "the specified :term:`BSON` type."
#~ msgstr ""

# 52569ba39ff84ad18b57ee178f5007b9
#~ msgid "Consider the following example:"
#~ msgstr ""

# 4e4c63aa9e7748578f331916f45a5446
#~ msgid ""
#~ "This query will select all documents "
#~ "in the ``inventory`` collection where "
#~ "the ``price`` field value is a "
#~ "Double."
#~ msgstr ""

# ff45173dff5946928563e5cce4f346bb
#~ msgid ""
#~ "If the ``field`` holds an array, "
#~ "the :query:`$type` operator performs the "
#~ "type check against the array elements"
#~ " and **not** the ``field``."
#~ msgstr ""

# 18f3fe2242c04653a3c197b9a4e8cfc9
#~ msgid "Consider the following example where the ``tags`` field holds an array:"
#~ msgstr ""

# 69692a695c5445968ecdd0f928a605bc
#~ msgid ""
#~ "This query will select all documents "
#~ "in the ``inventory`` collection where "
#~ "the ``tags`` array contains an element"
#~ " that is itself an array."
#~ msgstr ""

# f4f40b49d0e14394aa5c00f3cf5779ac
#~ msgid ""
#~ "If instead you want to determine "
#~ "whether the ``tags`` field is an "
#~ "array type, use the :query:`$where` "
#~ "operator:"
#~ msgstr ""

# d4bb9a3a13814faea51c81874d9c599d
#~ msgid "See the :issue:`SERVER-1475` for more information about the array type."
#~ msgstr ""

# c062fd7f91c345f8a86711a5cd243b67
#~ msgid ""
#~ "Refer to the following table for "
#~ "the available :term:`BSON` types and "
#~ "their corresponding numbers."
#~ msgstr ""

# 6ba1a306440a4f7e9d2d2b81e121da6d
#~ msgid "**Type**"
#~ msgstr ""

# b3ae2e06af2541939888815932afa5e6
#~ msgid "**Number**"
#~ msgstr ""

# 897f67d9539646118c5dfafa5792684f
#~ msgid "Undefined (deprecated)"
#~ msgstr ""

# ce6fcf68778442aa9f9d2a421a8d6e3e
#~ msgid "Object id"
#~ msgstr ""

# 676c2639fd2c4ae69348b0efa9cd9ab6
#~ msgid "255"
#~ msgstr ""

# 08c41e855d9448108a6d320e819489e1
#~ msgid ""
#~ "``MinKey`` and ``MaxKey`` compare less "
#~ "than and greater than all other "
#~ "possible :term:`BSON` element values, "
#~ "respectively, and exist primarily for "
#~ "internal use."
#~ msgstr ""

# de71905419aa4af99b798d143f24ef2b
#~ msgid ""
#~ "To query if a field value is "
#~ "a ``MinKey``, you must use the "
#~ ":query:`$type` with ``-1`` as in the "
#~ "following example:"
#~ msgstr ""

# 6083273e1aec4e8096988088dbc552a6
#~ msgid "Example"
#~ msgstr ""

# 52562d3d349d49e88be5b501287ab5ea
#~ msgid ""
#~ "Consider the following example operation "
#~ "sequence that demonstrates both type "
#~ "comparison *and* the special ``MinKey`` "
#~ "and ``MaxKey`` values:"
#~ msgstr ""

# 63b729f5015e45a7a07a889e334bc950
#~ msgid ""
#~ "To query for the minimum value of"
#~ " a :term:`shard key` of a "
#~ ":term:`sharded cluster`, use the following "
#~ "operation when connected to the "
#~ ":program:`mongos`:"
#~ msgstr ""

# 6811d8ab85d64507943ac590ef13e00d
#~ msgid ""
#~ "Storing values of the different types"
#~ " in the same field in a "
#~ "collection is *strongly* discouraged."
#~ msgstr ""

# 2c1e59c3232e4d0585ebb4b075413994
#~ msgid ""
#~ ":method:`~db.collection.find()`, "
#~ ":method:`~db.collection.insert()`, :query:`$where`, "
#~ ":term:`BSON`, :term:`shard key`, :term:`sharded "
#~ "cluster` ."
#~ msgstr ""

