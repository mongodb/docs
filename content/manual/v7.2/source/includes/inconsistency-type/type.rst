
Type of inconsistency document.  Returns a value of |incon-type| for this   
inconsistency type.


