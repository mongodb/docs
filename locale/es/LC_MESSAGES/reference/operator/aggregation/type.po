# SOME DESCRIPTIVE TITLE.
# Copyright (C) 2008-2016
# This file is distributed under the same license as the mongodb-manual
# package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2019.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: mongodb-manual 3.4\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2016-12-08 12:02-0500\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 2.6.0\n"

# d4bbf988f7974f2483b5728ffd72c3bf
#: ../source/reference/operator/aggregation/type.txt:3
msgid "$type (aggregation)"
msgstr ""

# b64bd8c1692c4eea99f174e663e70172
#: ../source/reference/operator/aggregation/type.txt
msgid "On this page"
msgstr ""

# e192009c6d8d4f9eb3dd791386a8525e
#: ../source/reference/operator/aggregation/type.txt:14
msgid "Definition"
msgstr ""

# 7d402f99fec8447ba373533442270be5
#: ../source/reference/operator/aggregation/type.txt:20
msgid ""
"Returns a string that specifies the :doc:`BSON type </reference/bson-"
"types/>` of the argument."
msgstr ""

# f945fd8c35f04e5aae6685fce1744a62
#: ../source/reference/operator/aggregation/type.txt:23
msgid ""
":expression:`$type` has the following :ref:`operator expression syntax "
"<agg-quick-ref-operator-expressions>`:"
msgstr ""

# 588f2206f51b43a5836b5161aa92cc0e
#: ../source/reference/operator/aggregation/type.txt:30
msgid "The argument can be any valid :ref:`expression <aggregation-expressions>`."
msgstr ""

# cc6ad411c27d468d97006b1a03f6cd20
#: ../source/reference/operator/aggregation/type.txt:34
msgid "Behavior"
msgstr ""

# b0cfa8014aac45ba812f016c4f0ae09c
#: ../source/reference/operator/aggregation/type.txt:36
msgid ""
"Unlike the :query:`$type` query operator, which matches array elements "
"based on their BSON type, the :expression:`$type` aggregation operator "
"does not examine array elements. Instead, when passed an array as its "
"argument, the ``$type`` aggregation operator returns the type of the "
"argument, i.e. ``\"array\"``."
msgstr ""

# 22f64d3123a3458a9400f3d2226e30b2
#: ../source/reference/operator/aggregation/type.txt:42
msgid ""
"If the argument is a field that is missing in the input document, "
":expression:`$type` returns the string ``\"missing\"``."
msgstr ""

# 3e4e567ed44a4eb99009492a9464a867
# bd517e8c1ac048f68984aa7b059b2a5e
#: ../source/reference/operator/aggregation/type.txt:53
#: ../source/reference/operator/aggregation/type.txt:89
msgid "Example"
msgstr ""

# 6bc0fb9c1be7482dbb2742b74c550a4d
#: ../source/reference/operator/aggregation/type.txt:54
msgid "Results"
msgstr ""

# ce53f805dce046ba9fadfbc807a17762
#: ../source/reference/operator/aggregation/type.txt:56
msgid "``{ $type: \"a\" }``"
msgstr ""

# eae4710260ac4f8e9ff0f2229752766c
#: ../source/reference/operator/aggregation/type.txt:57
msgid "``\"string\"``"
msgstr ""

# 66f62df1c9a14c01848db22f2c103136
#: ../source/reference/operator/aggregation/type.txt:59
msgid "``{ $type: /a/ }``"
msgstr ""

# 780556e0e1554dedbe1327da6ba07595
#: ../source/reference/operator/aggregation/type.txt:60
msgid "``\"regex\"``"
msgstr ""

# 4c60f0da177842f18367b5c7559507a6
#: ../source/reference/operator/aggregation/type.txt:62
msgid "``{ $type: 1 }``"
msgstr ""

# 1d3a927d71a94a06b4915840248d7a47
#: ../source/reference/operator/aggregation/type.txt:63
msgid "``\"double\"``"
msgstr ""

# fbd0acd7b10d45f4bcadf4b541747e2a
#: ../source/reference/operator/aggregation/type.txt:65
msgid "``{ $type: NumberLong(627) }``"
msgstr ""

# 0a5784fcb9e84d30b51cccab482a9a56
#: ../source/reference/operator/aggregation/type.txt:66
msgid "``\"long\"``"
msgstr ""

# 1f9af95d830c46c98a56437aafde2fe4
#: ../source/reference/operator/aggregation/type.txt:68
msgid "``{ $type: { x: 1 } }``"
msgstr ""

# d31c5584126f4fca8051223ac97ba04a
#: ../source/reference/operator/aggregation/type.txt:69
msgid "``\"object\"``"
msgstr ""

# 7246de5f63514059b431ce73400c951c
#: ../source/reference/operator/aggregation/type.txt:71
msgid "``{ $type: [ [ 1, 2, 3 ] ] }``"
msgstr ""

# b10cb37f321a45aca88ef60ed5c296ef
#: ../source/reference/operator/aggregation/type.txt:72
msgid "``\"array\"``"
msgstr ""

# eb2195ed901d4c3897c761dbca618177
#: ../source/reference/operator/aggregation/type.txt:76
msgid ""
"In the case of a literal array such as ``[ 1, 2, 3 ]``, enclose the "
"expression in an outer set of array brackets to prevent MongoDB from "
"parsing ``[ 1, 2, 3 ]`` as an :ref:`argument list <agg-quick-ref-"
"operator-expressions>` with three arguments (``1, 2, 3``). Wrapping the "
"array ``[ 1, 2, 3 ]`` in a :expression:`$literal` expression achieves the"
" same result."
msgstr ""

# 2eb5e505de2240d388cc7fdb1ac7adea
#: ../source/reference/operator/aggregation/type.txt:85
msgid ""
"See :ref:`operator expression syntax forms <agg-quick-ref-operator-"
"expressions>` for more information."
msgstr ""

# 4234818fa20a46328f7e67ebc05f2177
#: ../source/reference/operator/aggregation/type.txt:91
msgid ""
"This example uses a :term:`collection <collection>` named ``coll`` with "
"the following :term:`documents <document>`:"
msgstr ""

# c0eeac8b6795478988661a9c885d1db7
#: ../source/reference/operator/aggregation/type.txt:103
msgid ""
"The following aggregation operation uses the ``$type`` operator to "
"display the type of field ``a`` for all documents as part of the "
":pipeline:`$project` stage."
msgstr ""

# e5ac12ef5757452eb4e348cde61611df
#: ../source/reference/operator/aggregation/type.txt:115
msgid "The operation returns the following:"
msgstr ""

