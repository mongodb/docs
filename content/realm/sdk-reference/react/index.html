<!doctype html>
<head>
  <meta http-equiv="refresh" content="0; url=https://mongodb.com/docs/realm-sdks/react/latest" />
</head>
<body>
  <p><a href="https://mongodb.com/docs/realm-sdks/react/latest">Redirect</a></p>
</body>
