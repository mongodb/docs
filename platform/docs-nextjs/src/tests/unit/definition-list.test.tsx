import DefinitionList from '@/components/definition-list';
import { render } from '@testing-library/react';
import type { DefinitionListNode } from '@/types/ast';

import mockData from '@/tests/data/definition-list.test.json';
const typedMockData = mockData as DefinitionListNode;

it('DefinitionList renders correctly', () => {
  const tree = render(<DefinitionList nodeChildren={typedMockData.children} />);
  expect(tree.asFragment()).toMatchSnapshot();
});
