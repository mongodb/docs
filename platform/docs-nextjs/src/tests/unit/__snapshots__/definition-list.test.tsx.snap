// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DefinitionList renders correctly 1`] = `
<DocumentFragment>
  .emotion-0 {
  font-size: 13px;
  line-height: 20px;
  display: inline;
  -webkit-transition: all 0.15s ease-in-out;
  transition: all 0.15s ease-in-out;
  border-radius: 3px;
  font-family: 'Source Code Pro',Menlo,monospace;
  line-height: 20px;
  background-color: #F9FBFA;
  border: 1px solid #E8EDEB;
  color: #1C2D38;
  white-space: nowrap;
  font-size: unset;
  display: inline;
  color: var(--font-color-primary);
  background: var(--background-color-secondary);
  word-wrap: break-word;
  white-space: unset;
}

.lg-ui-0000:hover>.emotion-0 {
  -webkit-text-decoration: none;
  text-decoration: none;
}

.lg-ui-0000:hover>.emotion-0 {
  box-shadow: 0 0 0 3px #E8EDEB;
  border: 1px solid #C1C7C6;
}

.lg-ui-0000:focus-visible>.emotion-0 {
  box-shadow: 0 0 0 2px #FFFFFF,0 0 0 4px #0498EC;
  border: 1px solid #016BF8;
}

a .emotion-0 {
  color: inherit;
}

.emotion-1 p:first-of-type {
  margin-top: 0!important;
}

<dl>
    <dt>
      <code
        class="emotion-0"
      >
        MongoDefaultPartitioner
      </code>
    </dt>
    <dd
      class="emotion-1"
    >
      <strong>
        Default
      </strong>
      . Wraps the MongoSamplePartitioner and provides
help for users of older versions of MongoDB.
    </dd>
  </dl>
</DocumentFragment>
`;
